src/connectors/bandcamp.js

```diff
@@ -109,4 +109,6 @@ Connector.getUniqueID = () => {
 	return null;
 };
 
+Connector.getAlbumArtist = () => $('span[itemprop="byArtist"]').text().trim() || null;
```

Comment:
The same thing about trim, but also useless fallback to null.

Reply:
Without trim I see

```
Web Scrobbler: {
  "track": "Proper Preaching",
  "artist": "Luca Musto",
  "album": "Proper Preaching EP",
  "albumArtist": " \n          Luca Musto\n          ",
  "uniqueID": null,
  "duration": null,
  "currentTime": null,
  "isPlaying": true,
  "trackArt": "https://f4.bcbits.com/img/a1481550943_16.jpg"
}
```

Reply:
Should getTextFromSelectors do trim?

Reply:
No, it should not.

albumArtist is not trimmed, because default filter is not modified to trim albumArtist.

Reply:
Ahh I'll update the filter